custom_dict.txt

```diff
@@ -660,6 +662,7 @@ LimitRanges
 LinkedIn
 Linkerd
 linter
```

Comment:
We can remove `linter` since it's covered by `linter(s)`

Reply:
The check was failing on `linter` specifically